timescaledb/how-to-guides/install-timescaledb/installation-cloud-image.md

```diff
@@ -0,0 +1,147 @@
+
+This AMI is available in the US and EU cloud regions. The current image IDs for
+each of the available regions are:
+
+|Region|Country or state|Image ID|
+|-|-|-|
+|us-east-1|North Virginia|`ami-0142e0ff4336460d8`|
+|us-east-2|Ohio|`ami-093758173e997a8aa`|
+|us-west-1|North California|`ami-0065769df35cdaf5e`|
+|us-west-2|Oregon|`ami-07a7e914792b3a760`|
+|eu-central-1|Germany|`ami-07bf96202903b78fd`|
+|eu-north-1|Sweden|`ami-03aa8f2270308356e`|
+|eu-west-1|Ireland|`ami-09d5dd90db4b4f9aa`|
+|eu-west-2|England|`ami-0a06dcf5597a26395`|
+|eu-west-3|France|`ami-00f44167b13c0a9ac`|
+
+<highlight type="note">
+This section shows how to use the AMI from within the AWS EC2 dashboard.
+However, you can also use the AMI to build an instance using tools like
+Cloudformation, Terraform, the AWS CLI, or any other AWS deployment tool that
+supports public AMIs.
+</highlight>
+
+<procedure>
+
+### Installing self-hosted TimescaleDB from a pre-build cloud image
+1.  Make sure you have an [Amazon Web Services account][aws-signup], and
+    are signed in to [your EC2 dashboard][aws-dashboard].
+1.  Navigate to `Images → AMIs`.
+1.  In the search bar, change the search to `Public images` and enter the
+    Image ID for your region. Alternatively, you can use the `Timescale` search
+    term to find all available Timescale images.
+1.  Select the image you want to use, and click `Launch`.
+    <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/aws_launch_ami.png" alt="Launch an AMI in AWS EC2"/>
+
+</procedure>
+
+When you have completed the installation, you need to configure your database so
+that you can use it. The easiest way to do this is to run the `timescaledb-tune`
+script, which is included with the `timescaledb-tools` package. For more
+information, see the [configuration][config] section.
+
+## Set up the TimeascaleDB extension
+When you have PostgreSQL and TimescaleDB installed, you can connect to it from
+your local system using the `psql` command-line utility. This is the same tool
+you might have used to connect to PostgreSQL before, but if you haven't
+installed it yet, check out our [installing psql][install-psql] section.
+
+<procedure>
+
+### Setting up the TimescaleDB extension
+1.  On your local system, at the command prompt, connect to the PostgreSQL
+    instance as the `postgres` superuser:
+    ```bash
+    psql -U postgres -h localhost
+    ```
+    If your connection is successful, you'll see a message like this, followed
+    by the `psql` prompt:
+    ```
+    psql (13.3, server 12.8 (Ubuntu 12.8-1.pgdg21.04+1))
+    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+    Type "help" for help.
+    tsdb=>
+    ```
+1.  At the `psql` prompt, create an empty database. Our database is
+    called `example`:
+    ```sql
+    CREATE database example;
+    ```
+1.  Connect to the database you created:
+    ```sql
+    \c tutorial
```

Comment:
shouldn't this be `example` since thats the database we created just before

Reply:
```suggestion
    \c example
```